src/main/java/com/github/hey_world_team/currency_converter/model/chain/Order.java

```diff
@@ -0,0 +1,19 @@
+package com.github.hey_world_team.currency_converter.model.chain;
+
+import lombok.*;
+import lombok.extern.slf4j.Slf4j;
+
+@Data
+@AllArgsConstructor
+@EqualsAndHashCode
+@ToString
+@Slf4j
+@Builder
+public class Order {
+    private long id;
+    private Trader trader;
+    private Trade trade;
+    private boolean isBuy;
```

Comment:
may be better to change toBuy and toSell

Reply:
yes it's a good idea, i will rename the field toBuy, and the method will be isToBuy